docs/admin-tasks.md

```diff
@@ -113,3 +113,16 @@ To see all the editions that were unpublished between two times run:
 ```
 bundle exec rake "csv_report:unpublishings_by_date_range['2020-10-01 10:00', '2020-12-31 10:00']"
 ```
+
+## Viewing the Sidekiq UI
+
+We have access to the Sidekiq UI but because Publishing API doesn't have a
+frontend we have to use port forwarding to see it in our live environments.
+
+To view the UI run:
+
+```
+kubectl -n apps port-forward deployment/publishing-api 8080:8080
```

Comment:
I get the following when running this locally, is there anything I need to do first?
```
$ kubectl -n apps port-forward deployment/publishing-api 8080:8080
E0812 08:02:18.886000   73169 memcache.go:238] couldn't get current server API group list: the server has asked for the client to provide credentials
E0812 08:02:19.827681   73169 memcache.go:238] couldn't get current server API group list: the server has asked for the client to provide credentials
E0812 08:02:20.647060   73169 memcache.go:238] couldn't get current server API group list: the server has asked for the client to provide credentials
E0812 08:02:21.473412   73169 memcache.go:238] couldn't get current server API group list: the server has asked for the client to provide credentials
error: You must be logged in to the server (the server has asked for the client to provide credentials)
```

Reply:
I've added some notes in the documentation about the steps needed to run these commands

Reply:
Great, thanks for adding this.